refarch-eai/pom.xml

```diff
@@ -99,6 +99,11 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<scope>test</scope>
+		</dependency>
```

Comment:
Could you please fix the formatting here?

Reply:
@DanielOber we could even think about including the apache commons dependency by default (not only as test deps), that's how we currently do it in the backend. What do you think about this?

Reply:
formatting done